fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2656,4 +2656,10 @@ public class Config extends ConfigBase {
 
     @ConfField(mutable = true)
     public static boolean use_lock_manager = false;
+
+    @ConfField(mutable = true)
+    public static boolean jdbc_meta_cache_enable = false;
+
+    @ConfField(mutable = true)
+    public static long jdbc_meta_cache_expire_sec = 600L;
```

Comment:
Jdbc metadata usually is stable, so we can enlarge expire time. And other metadata cache' configuration usually use session variable.

Reply:
600 seconds is an empirical value. From a practical perspective, the metadata of MySQL may not be as stable. This is an empirical value we obtained from other projects that falls between performance and cache time. I think we can give it a try first. If there are good suggestions from customers, we can make the necessary changes.